### ISSUE TITLE
Building v10-project-base-spring fails

### ISSUE THREAD

**Issue by khmarbaise** [vaadin/testbench#1096]
I have created a [starter app](https://vaadin.com/start#vaadin10) and  selected https://vaadin.com/start/v10-project-base-spring and downloaded the generated zip and tried to build it which results in:
```
INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] Non-resolvable import POM: Failure to find com.vaadin:vaadin-testbench-bom:pom:6.0.0 in http://localhost:8081/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of nexus has elapsed or updates are forced @ com.vaadin:vaadin-bom:[unknown-version], /home/khm/.m2/repository/com/vaadin/vaadin-bom/10.0.1/vaadin-bom-10.0.1.pom, line 245, column 25
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project de.xxx:ui:1.0-SNAPSHOT (/home/khm/../gui/pom.xml) has 1 error
[ERROR]     Non-resolvable import POM: Failure to find com.vaadin:vaadin-testbench-bom:pom:6.0.0 in http://localhost:8081/nexus/content/groups/public/ was cached in the local repository, resolution will not be reattempted until the update interval of nexus has elapsed or updates are forced @ com.vaadin:vaadin-bom:[unknown-version], /home/khm/.m2/repository/com/vaadin/vaadin-bom/10.0.1/vaadin-bom-10.0.1.pom, line 245, column 25 -> [Help 2]
```

After some more investigation the result is the the vaadin-testbench-bom in version 6.0.0 is not in [central](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.vaadin%22%20AND%20a%3A%22vaadin-testbench-bom%22%20AND%20v%3A%226.0.0%22) available. which is referenced in the vaadin-bom:10.0.1...

The version 6.0.1 of vaadin-testbench-bom is available in [Central.](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22com.vaadin%22%20AND%20a%3A%22vaadin-testbench-bom%22)

Apart from that I haven't found the version in any known repositories...Nor have I found a hint how to configure Maven to handle this correctly ? Or do i miss something?

**Comment by alvarezguille** [comment]
Hi, sorry for the confusion. 

Testbench 6.0.0 is available in `vaadin-addons` repository
```
<repository>
   <id>vaadin-addons</id>
   <url>http://maven.vaadin.com/vaadin-addons</url>
</repository>
```

Testbench 6.0.1+ is available in maven central repository. A new version of Vaadin is about to be released without the need to have `vaadin-addons` repo. Initially reported in vaadin/platform#216

**Comment by khmarbaise** [comment]
First thanks for your answer... So after adding the `vaadin-addons` to my repository manager I get it to be build. And I can encourage you to follow the suggestions done in https://github.com/vaadin/platform/issues/216 to make the initial startup easier...

**Comment by quickstep24** [comment]
I have the same problem, but adding ´vaadin-addons´ did not fix it for me:

```
Caused by: org.gradle.api.internal.artifacts.ivyservice.ivyresolve.parser.MetaDataParseException: Could not parse POM https://dl.bintray.com/mvysny/github/com/github/vok/karibudsl/karibu-dsl-v10/0.4.6/karibu-dsl-v10-0.4.6.pom
Caused by: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve com.vaadin:vaadin-bom:10.0.1.
Caused by: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve com.vaadin:vaadin-bom:10.0.1.
Caused by: org.gradle.api.internal.artifacts.ivyservice.ivyresolve.parser.MetaDataParseException: Could not parse POM https://plugins.gradle.org/m2/com/vaadin/vaadin-bom/10.0.1/vaadin-bom-10.0.1.pom
Caused by: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve com.vaadin:vaadin-testbench-bom:6.0.0.
Caused by: org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve com.vaadin:vaadin-testbench-bom:6.0.0.
Caused by: org.gradle.api.resources.ResourceException: Could not get resource 'https://maven.vaadin.com/vaadin-addons/com/vaadin/vaadin-testbench-bom/6.0.0/vaadin-testbench-bom-6.0.0.pom'.
Caused by: org.gradle.internal.resource.transport.http.HttpErrorStatusCodeException: Could not HEAD 'https://maven.vaadin.com/vaadin-addons/com/vaadin/vaadin-testbench-bom/6.0.0/vaadin-testbench-bom-6.0.0.pom'. Received status code 403 from server: Forbidden
```

Is there any reason why testbench is on a separate repository?

**Comment by alvarezguille** [comment]
There might've been some temporary outage causing:
>Caused by: org.gradle.api.resources.ResourceException: Could not get resource 'https://maven.vaadin.com/vaadin-addons/com/vaadin/vaadin-testbench-bom/6.0.0/vaadin-testbench-bom-6.0.0.pom'.

I tried accessing https://maven.vaadin.com/vaadin-addons/com/vaadin/vaadin-testbench-bom/6.0.0/vaadin-testbench-bom-6.0.0.pom now and it works as expected.

> Is there any reason why testbench is on a separate repository?

Originally all Vaadin official add-ons were in vaadin-addons repository, we now have newer versions in maven central.

A new version that doesn't require vaadin-addons repo is on its way. I will close this issue once it's released and available.

**Comment by quickstep24** [comment]
I am using vaadin gradle plugin "com.devsoap.vaadin-flow" version "1.0.0.M3.1".
Reported error is 
```
Could not HEAD 'https://maven.vaadin.com/vaadin-prereleases/com/vaadin/vaadin-testbench-bom/6.0.0/vaadin-testbench-bom-6.0.0.pom'. Received status code 403 from server: Forbidden
```
The log also shows a number of
```
NEGOTIATE authentication error: No valid credentials provided (Mechanism level: No valid credentials provided (Mechanism level: Failed to find any Kerberos tgt))
```
I am behind a firewall with a proxy configuration and other modules do get through

**Comment by alvarezguille** [comment]
[Vaadin 10.0.2](https://github.com/vaadin/platform/releases/tag/10.0.2) has been released and all artifacts should be resolvable from maven central. Please try with the new version.
New versions of starters available in vaadin.com/start will also be released using 10.0.2, but for projects already created you'll need to update the vaadin.version property.
Please re-open if you still have issues related to this.
